src/utils/convert.ts

```diff
@@ -1321,6 +1383,218 @@ async function processM2NRehypeImageNodes(
   }
 }
 
+function getN2LRehypeHeaderNodes(rehype: HRoot) {
```

Comment:
Since this file is already super long, we should consider separating it into two file (e.g. a `convertMD.ts` and a `convertLatex.ts`). For now, we can just let it be as it is and I can do this refactor later.